packages/battery/ios/battery.podspec

```diff
@@ -12,8 +12,8 @@ Flutter plugin for accessing information about the battery.
   s.license          = { :file => '../LICENSE' }
   s.author           = { 'Flutter Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :path => '.' }
-  s.source_files = 'Classes/**/*'
```

Comment:
Are we sure that the `.h` files are not considered source files?
https://guides.cocoapods.org/syntax/podspec.html#source_files suggests differently.